lib/StoryEngine/DataLayer/model/StoryDataModel.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
+namespace StoryEngine.StoryEngineDataModel;
+
+public enum PrioritizationTypeDataModel
+{
+    strawManRandom,
+    sumOfCategoryMaximums,
+    eventBased,
+    bestObjectiveFunction
```

Comment:
These are the different prioritization algorithms for the engine?! 😮

Reply:
Haha yeah, though I may not implement them all in this version, in which case I could remove this. In the last paper we attempted, we were comparing the different algorithms to see which we thought would be better.